runtime/src/configs/mod.rs

```diff
@@ -128,7 +153,7 @@ impl frame_system::Config for Runtime {
 impl pallet_timestamp::Config for Runtime {
     type Moment = u64;
     type OnTimestampSet = Aura;
-    type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;
+    type MinimumPeriod = ConstU64<0>;
```

Comment:
Do you know the implications of this?

Reply:
Basically now that elastic scaling is implemented we don't have a minimum time between blocks